### PR TITLE
Form feature

### PR DESCRIPTION
**Pull request by Eshy10**
## HTML FORM

In this project:
1. We created a left floating login button
2. We created a validated sign up form
3. We made the website responsive

### CODE REVIEW

**Review by fatymahmed — COMMENTED** [review]
## Hi @Eshy10 @nyatindopatrick,
First of all, your project is very impressive 👏👏 You have a descriptive README file and no HTML markup errors, great job.

You just need to make minor changes to make it look better.
**Intuit products**

- [ ]  It shows the Mint logo and text (logo on the left, text on the right)
- [ ] It shows the Quickbooks logo and text (logo on the left, text on the right)
- [ ] It shows the Turbotax logo and text (logo on the left, text on the right)
- I noticed you used a screenshot which contains all 4 logos and the text in it, please change this and add each separately and also add the text by yourself.

-  For the phone input, it should be of type `tel`, please change this.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        </div>
+        <div class="login-form">
+            <h2>Create an Intuit account</h2>
+            <p>One account for everything Intuit, including Mint.</p>
+            <a href="#">Learn more</a>
+            <form class="form-items">
+                <label for="email">Email Address</label>
+                <input type="email" id="email">
+                <label for="phone">Phone (recommended)</label>
+                <input type="number" id="phone">
```
Comment (fatymahmed) [inline index.html]: - Change type of this input to `tel`

**Review by fatymahmed — CHANGES_REQUESTED** [review]
### Kindly make the suggested changes and request for another code review.

## Status: Changes Required
_Feel free to reach out to me on slack (fatima) if you have any questions, I'd be happy to help_

**Review by adriaanbd — APPROVED** [review]
Congrats, you've met all requirements for this project.

Status: `Approved`
Reviewer: `Adriaan Beiertz1

[`OPTIONAL`]. You might want to consider working on the alignment of the logos and text above the form because it doesn't look centered, in comparison to the original:

image

**Pull request merged**